recipe/meta.yaml

```diff
@@ -78,6 +86,7 @@ outputs:
         - pkg-config  # [not win]
         - unzip  # [not win]
         - make  # [not win]
+        - sed  # [osx]
```

Comment:
If only the `build-shared.sh` has the `sed` workaround, is this needed for `libprotobuf-static`?

Reply:
`sed` workaround is unnecessary for static builds, removed.